packages/styles/scss/patterns/sub-patterns/card/index.scss

```diff
@@ -62,6 +75,14 @@
       transition: $duration--moderate-01 motion(standard, productive);
     }
 
+    &--link {
+      height: 100%;
```

Comment:
Sorry, help me understand what this is here for?

Reply:
this helped to fix that locale modal issue where the card wrapper wasn't taking the full height of the card

Reply:
Ok, I feel like that should be in apart of the local selector instead. We don't necessarily want all of our cards to be height 100% do we?

Reply:
ok done